lineman/app/components/pie_chart/pie_chart.scss

```diff
@@ -0,0 +1,19 @@
+.pie-chart--big{
```

Comment:
-- means modifier
__ means element

In this case size is a modifier of the block rather than an element within it.